manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -74,15 +74,15 @@ type ManageIQSpec struct {
 	// +optional
 	OIDCCACertSecret string `json:"oidcCaCertSecret"`
 	// URL for OIDC authentication introspection
-	// Only used with the openid-connect authentication type
-	// +optional
-	OIDCOAuthIntrospectionURL string `json:"oidcAuthIntrospectionURL"`
-	// Secret name containing the OIDC client id and secret
 	// Only used with the openid-connect authentication type.
 	// If not specified, the operator will attempt to fetch its value from the
 	// "token_introspection_endpoint" field in the Provider metadata at the
 	// OIDCProviderURL provided.
 	// +optional
+	OIDCOAuthIntrospectionURL string `json:"oidcAuthIntrospectionURL"`
+	// Secret name containing the OIDC client id and secret
+	// Only used with the openid-connect authentication type
```

Comment:
why is there a diff here ?

Reply:
Because https://github.com/ManageIQ/manageiq-pods/pull/582 updated the description at wrong places.

In the original PR for master branch (https://github.com/ManageIQ/manageiq-pods/pull/571), `oidcAuthIntrospectionURL` wad updated.  But in the Jansa specific PR (https://github.com/ManageIQ/manageiq-pods/pull/582), `oidcClientSecret` was updated instead.

I noticed this because backport of https://github.com/ManageIQ/manageiq-pods/pull/606 conflicted.